include/Bezier/bezier.h

```diff
@@ -278,7 +278,7 @@ class Curve
    * \param locked_curve Curve on which calculation are based.
    * \param beta_coeffs Beta-constraints used to calculate continuity. Size defines continuity order.
    */
-  void applyContinuity(const Curve& source_curve, const std::vector<double>& beta_coeffs);
+  void applyContinuity(const Curve& curve, const std::vector<double>& beta_coeffs);
```

Comment:
`source_curve` may not be the most intuitive name, but some attribute describing the relationship/purpose of this curve would improve readability. Just `curve` is not really descriptive. How about `target`?

Also, the parameter name in the comment must be updated.